Tests/AzurePSDrive.Tests.ps1

```diff
@@ -7,10 +7,12 @@ param (
     [string]$subscriptionName = 'AutomationTeam'
 )
 
+function New-PartialGuidForName { ([guid]::NewGuid().ToString() -replace '-','')[0..9] -join '' }
+
 #region Script variables
-$resourceGroupName = 'AzurePSDrive.Test'
+$resourceGroupName = "rgapsdt$(New-PartialGuidForName)"
```

Comment:
Just curious, what are rgapsdt and saapsdt and where are they defined?

Reply:
@jianyunt I simply picked prefixes that are related to the resources. The second half of the names are generated. 'rgapsdt' is Resource Group Azure PS Drive Test. 'saapsdt' is Storage Account Azure PS Drive Test. For testing, the names themselves don't matter. All that matters is they are unique.